tests/unit/test_notebooks.py

```diff
@@ -52,17 +56,46 @@ def test_optimize_criteo(tmpdir):
     os.environ["INPUT_DATA_DIR"] = str(tmpdir)
     os.environ["OUTPUT_DATA_DIR"] = str(tmpdir)
 
-    notebook_path = os.path.join(dirname(TEST_PATH), "examples", "optimize_criteo.ipynb")
+    notebook_path = os.path.join(
+        dirname(TEST_PATH),
+        "examples/scaling-criteo/",
+        "01-Download-Convert.ipynb",
+    )
     _run_notebook(tmpdir, notebook_path)
 
 
+def test_movielens_example(tmpdir):
```

Comment:
I like having a movielens test!  But I think we should only be doing the full download/runs in the integration tests -and should just use 'mock' data like we are for criteo / rossmann.  Also The integration test should rely on a precached vesion of the data, rather than downloading the dataset here

Also this is failing w/ 
```
Traceback (most recent call last):
  File "/tmp/pytest-of-jenkins/pytest-0/test_movielens_example0/notebook.py", line 119, in <module>
    tf.keras.utils.plot_model(model)
  File "/opt/conda/envs/rapids/lib/python3.8/site-packages/tensorflow/python/keras/utils/vis_utils.py", line 325, in plot_model
    dot = model_to_dot(
  File "/opt/conda/envs/rapids/lib/python3.8/site-packages/tensorflow/python/keras/utils/vis_utils.py", line 117, in model_to_dot
    raise ImportError(message)
ImportError: ('You must install pydot (`pip install pydot`) and install graphviz (see instructions at https://graphviz.gitlab.io/download/) ', 'for plot_model/model_to_dot to work.')
```
probably because the CI container doesn't have the right packages installed for this . Lets disable for now and come back to this soon
```suggestion
@pytest.mark.skip(reason="Need to install pydot / use mock data on this")
def test_movielens_example(tmpdir):
```